packages/server/postgres/migrations/1712074131388_priority.ts

```diff
@@ -0,0 +1,32 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    ALTER TABLE "EmbeddingsJobQueue"
+      ALTER COLUMN "priority" TYPE INTEGER;
+    ALTER TABLE "EmbeddingsJobQueue"
+      ADD COLUMN "model" VARCHAR(255),
+      ADD COLUMN "embeddingsMetadataId" INTEGER,
```

Comment:
i added this & model back in because the foreign key is nice to have for debugging. writing SQL against JSON is just a drag